Review

Tufahel opened the pull request:
- A free code camp project.

Tufahel committed:
bable, html, css and js file added

Tufahel committed:
licence, readme added

abdulhamiid reviewed (changes requested):
Hi @Tufahel, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 

## Invalid Code Review Request

You have submitted a project, but you did not make a review at all, I can only see an opened pull request.

According to the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md), you are required to **Copy your code from the project that you completed during the Admissions Trial** and then review it yourself.

The main goal of the exercise is that you perform a code review, and I will make a review based on how well you performed the code review exercise. :blush: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

Tufahel commented on README.md:
```diff
@@ -1 +1,5 @@
-# cr-review-previous-project
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
It's nice you are using README.md.
It would be better if your README were more descriptive. You can follow this sample as an example : [LINK](https://github.com/Kingstalux/React-Capstone/blob/dev/README.md)

abdulhamiid replied:
- Please consider providing the student with the official [Microverse README Template
](https://github.com/microverseinc/readme-template) because it provided students with **descriptive details** on what to be included in every section of the [README](https://github.com/microverseinc/readme-template) and that will make it easier for them to use and customize.

Tufahel commented on css/style.css:
```diff
@@ -0,0 +1,75 @@
+    color: #5a5;
+    text-decoration: none;
+}
+
+/* button:focus {
+    background-color: #7a7;
+} */
```
You have done good works with CSS.
If you remove the comment section, it would be more eye-catchy.

abdulhamiid replied:
Great job on pointing out issues on commented code :+1:

Tufahel reviewed (commented):
I have added my reviews. Give me feedback what should I improve to give a review.## 🟡 Status: Changes Required 🛑 

### Hi @Tufahel   👋🏻, 

Your have done a great job :+1: . You are almost done! :rocket: 
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media3.giphy.com/media/3ogwFKH8NPzxxIl2Cc/giphy.gif?cid=ecf05e47obp5ckpje3sx3xfz81akt1uu1sbf6mp5hzhrjtkm&rid=giphy.gif&ct=g)

## To Highlight 👏🏻

:heavy_check_mark: Project is working fine.
:heavy_check_mark: Commit are descriptive

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Tufahel commented:
> Hi @Tufahel,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, but you did not make a review at all, I can only see an opened pull request.
> 
> According to the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md), you are required to **Copy your code from the project that you completed during the Admissions Trial** and then review it yourself.
> 
> The main goal of the exercise is that you perform a code review, and I will make a review based on how well you performed the code review exercise. blush
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

Thanks for your review. I have fixed this issue. and added my reviews.

abdulhamiid commented on README.md:
```diff
@@ -1 +1,5 @@
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
- Please consider providing the student with the official [Microverse README Template
](https://github.com/microverseinc/readme-template) because it provided students with **descriptive details** on what to be included in every section of the [README](https://github.com/microverseinc/readme-template) and that will make it easier for them to use and customize.

abdulhamiid commented on css/style.css:
```diff
@@ -0,0 +1,75 @@
+    color: #5a5;
+    text-decoration: none;
+}
+
+/* button:focus {
+    background-color: #7a7;
+} */
```
Great job on pointing out issues on commented code :+1:

abdulhamiid reviewed (changes requested):
# Status: Required Changes :recycle:
Hi @Tufahel, 
![](https://media.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif)
Good job so far providing feedback with GitHub features and communicating professionally on the code you have reviewed! :blush: 

However, there are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 

## Highlights 🌟

- ✔️ Requested changes are very descriptive
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

[OPTIONAL ] - Kindly consider setting up linters for this repository
- Adding code adding linters correctly for this repository, code linters help to diagnose and fix technical issues—e.g. code smells—in the code.
     - Please kindly visit the ***html-css*** in the [Linters Config](https://github.com/microverseinc/linters-config) repo to learn how to set up linters
 - :green_circle: Check linters:
    - Linter should be set up correctly for this repository.
    - Linter result should be green :heavy_check_mark: . AKA: Linter should report NO errors.
    - There should be no custom changes to linter config (as an attempt to cheat linter error reports).


Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
